bin.src/make_latiss_export.py

```diff
@@ -52,40 +63,66 @@ def main():
     args = _make_parser().parse_args()
     src_butler = daf_butler.Butler(args.src_repo)
 
+    if args.target_repo:
+        target_butler = daf_butler.Butler(args.target_repo, writeable=False)
+    else:
+        # If no target_butler ig given, create an empty one.
+        temp_dir = tempfile.TemporaryDirectory()
+        config = daf_butler.Butler.makeRepo(temp_dir.name)
+        target_butler = daf_butler.Butler(config)
+
     with time_this(msg="Exporting datasets and collections", level=logging.INFO):
-        _export_for_copy(src_butler)
+        _export_for_copy(src_butler, target_butler)
+
+    if not args.target_repo:
+        temp_dir.cleanup()
 
 
-def _export_for_copy(butler):
+def _export_for_copy(butler, target_butler):
     """Export selected data to make copies in another butler repo.
 
     Parameters
     ----------
     butler : `lsst.daf.butler.Butler`
         The source Butler from which datasets are exported.
+    target_butler : `lsst.daf.butler.Butler`
+        The target Butler to which datasets are exported. If given,
+        it is checked to avoid exporting existing datasets. No
+        checks are done to verify if datasets are really identical.
```

Comment:
I think this description was written when `target_butler` was still optional.

Reply:
Corrected